setup_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -o errexit
+
+if [[ -z "${MININORCAL_OSM_PATH}" || -z "${MININORCAL_GTFS_PATH}" ]]; then
```

Comment:
script expects gcs paths, correct? Would be good to add a comment about that if so

Reply:
Yup - added one, good call!